src/Entity/DataField.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Dotmailer\Entity;
+
+class DataField implements Arrayable
```

Comment:
Is this now useful for the Contact entity?

Reply:
Unfortunately the DataField in the Contact entity is different - it is a simple key/value object, rather than a distinct entity. The DataField entity conversely doesn't store actual value, rather just the definition